libs/process/opts.go

```diff
@@ -0,0 +1,66 @@
+package process
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os/exec"
+
+	"github.com/databricks/cli/libs/env"
+)
+
+type execOption func(context.Context, *exec.Cmd) error
+
+func WithEnv(key, value string) execOption {
+	return func(ctx context.Context, c *exec.Cmd) error {
+		// we pull the env through lib/env such that we can run
+		// parallel tests with anything using libs/process.
+		if c.Env == nil {
+			for k, v := range env.All(ctx) {
+				c.Env = append(c.Env, fmt.Sprintf("%s=%s", k, v))
+			}
+		}
```

Comment:
Thinking more about this; this initialization should happen regardless of this option.